site/cni-plugin.md

```diff
@@ -1,13 +1,14 @@
 ---
-title: Integrating Kubernetes via the CNI Plugin
+title: Weave CNI Plugin
```

Comment:
'Integrating Kubernetes and Mesos via the CNI Plugin' is fine with me, if you'd like to keep the same structure.  Regarding the action rule.  It's not an all in rule, IMO.  Normally you'd want the headings to be action-orientated if the user is required to do something.  But if it's straight up description or an explanation, then I'd say a regular title is just fine.  Sometimes a document might have a combination of these titles.